benchmark_runner/common/ocp_resources/create_ocp_resource.py

```diff
@@ -77,23 +77,23 @@ def create_resource(self, resource: str):
         self.remove_resource_files(path=os.path.join(self.__dir_path, resource))
         resource_files = self.get_sorted_resources(resource=resource)
         if 'lso' == resource:
-            self.__create_lso = CreateLSO(self.__oc, path=os.path.join(self.__dir_path, resource), resource_list=resource_files)
-            self.__create_lso.create_lso()
+            create_lso = CreateLSO(self.__oc, path=os.path.join(self.__dir_path, resource), resource_list=resource_files)
+            create_lso.create_lso()
```

Comment:
What's the issue with this?

Reply:
No need to create class object if you are not using it in several places